src/main/java/com/jw/error/GlobalExceptionHandler.java

```diff
@@ -53,6 +53,10 @@ private Response handleGenericException(Throwable exception) {
     }
 
     @Override
+    //REVIEW-VINI: Is there a way that we can remove all these if-else on this method? Maybe you can try to use
+    // https://javadoc.io/doc/io.quarkus.resteasy.reactive/resteasy-reactive/3.16.3/org/jboss/resteasy/reactive/server/ServerExceptionMapper.html
+    // instead of this method, more details in here https://quarkus.io/guides/rest#exception-mapping
+    // I'm afraid of how much this method can grow in case more exception will be created
```

Comment:
I changed to use @ ServerExceptionMapper, now there is no need for this method, thanks for that proposition!